warrior.siml

```diff
@@ -2,7 +2,7 @@
 # - is used for comment line
 
 debug=1
-slowmotion=0 # Simulation in real time
+slowmotion=21 # Simulation in real time
```

Comment:
you kinda missunderstood slowmotion variable

it is boolean value 0 / 1 which will play simulation in real time.